CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Fix #3679: output additionalProperties field with correct value type for map-like fields (CRD Generator)
 * Fix #3648: `Serialization.unmarshal` fails to deserialize YAML with single document in presence of document delimiter(`---`)
 * Fix #3568: Pod file upload fails if the path is `/`
+* Fix #3745: updating require logic to account for item namespace
```

Comment:
I think this got mixed up with another PR

Reply:
sorry, yes that was from the change specifically for require.  updated now.